app/presenters/organisation_presenter.rb

```diff
@@ -17,10 +17,15 @@ def organisation_type
     I18n.t("organisation.organisation_type.#{super}")
   end
 
-  def filename_for_activities_template
+  def filename_for_activities_template(type:)
+    human_readable_type = {
+      ispf_oda: "ISPF_ODA",
+      non_ispf: "GCRF_NF_OODA"
+    }.fetch(type, "").tr(" ", "_")
```

Comment:
@Gweaton this `human_readable_type` will be reused elsewhere for Level C/D, but we could refactor when it comes to that or just repeat it, given it's a small bit of code?